README.md

```diff
@@ -12,85 +12,141 @@ providers, GCE has a couple of advantages for Chef cookbook testing:
-(e.g. 123456789012@developer.gserviceaccount.com - not the project
-owner's email address).
-
-If you are not using the `public_key_path` setting (see below) and
-have not [set up SSH keys for your GCE
-environment](https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys),
-you must also do that prior to using kitchen-gce.  Also, you will
-likely want to add your SSH keys to ssh-agent prior to converging any
-instances.
+"project id".
+
+### Authentication and Authorization
+
+The [underlying API](https://github.com/google/google-api-ruby-client) this plugin uses relies on the 
+[Google Auth Library](https://github.com/google/google-auth-library-ruby) to handle authentication to the
+Google Cloud API. The auth library expects that there is a JSON credentials file located at:
+
+`~/.config/gcloud/application_default_credentials.json`
+
+The easiest way to create this is to download and install the [Google Cloud SDK](https://cloud.google.com/sdk/) and run the
+`gcloud auth login` command which will create the credentials file for you.
+
+If you already have a file you'd like to use that is in a different location, set the
+`GOOGLE_APPLICATION_CREDENTIALS` environment variable with the full path to that file.
+
+### SSH Keys
+
+In order to bootstrap Linux nodes, you will first need to ensure your SSH
+keys are set up correctly. Ensure your SSH public key is properly entered 
+into your project's Metadata tab in the GCP Console. GCE will add your key
+to the appropriate user's `~/.ssh/authorized_keys` file when Chef first
+connects to perform the bootstrap process.
+
+ * If you don't have one, create a key using `ssh-keygen`
+ * Log in to the GCP console, select your project, go to Compute Engine, and go to the Metadata tab.
+ * Select the "SSH Keys" tab.
+ * Add a new item, and paste in your public key.
+    * Note: to change the username automatically detected for the key, prefix your key with the username
+      you plan to use to connect to a new instance. For example, if you plan to connect
+      as "chefuser", your key should look like: `chefuser:ssh-rsa AAAAB3N...`
+ * Click "Save".
+
+You can find [more information on configuring SSH keys](https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys) in
+the Google Compute Engine documentation.
 
 ## Installation
 
-Assuming you are using Bundler, ensure the Gemfile within your Chef
-cookbook contains at least the following:
+Install the gem with:
 
-```ruby
-source 'https://rubygems.org'
+```sh
+gem install kitchen-google
+```
+
+Or, even better, if you're using the ChefDK:
 
-gem 'berkshelf'
+```sh
+chef gem install kitchen-google
+```
+
+If you're using Bundler, simply add it to your Gemfile:
 
-group :integration do
-  gem 'kitchen-gce'
-end
+```ruby
+gem "kitchen-google", "~> 1.0"
 ```
 
-Then, execute `bundle install`.
+... and then run `bundle install`.
+
 
 ## Configuration
 
-### `area`
+### `project`
```

Comment:
s/project/project_id/

I think `project_id` reinforces your point that about id vs name.